Large number of changes and improvements

Jason-Morcos opened the pull request:
This pull request makes many changes and improvements from adding support for favorites, history, and a home page to adding a loading indicator and adding error messages. The README file indicates many of the changes made.

steventroughtonsmith committed:
Initial checkin

steventroughtonsmith committed:
Cleanup

steventroughtonsmith committed:
More Cleanup

Commit:
Improved cursor handling

Improvements in movement and activation of the cursor, for use on real
hardware. Activation of cursor is done through a click on the remote
trackpad.

jvanakker committed:
Update README.mdown

jvanakker committed:
Update README.mdown

Commit:
added a script file for automatically changing the Availability.h

the script changes both the simulator and device Availability.h

Jason-Morcos committed:
Added some automatic switch to search functionality

Jason-Morcos committed:
Many, many improvements including a loading indicator

Jason-Morcos committed:
Added support for text entry to text fields

Also, several other changes including changing the switch from
scrolling to clicking to a double press of the touch surface of the
remote. Added support for a home page and added a menu that shows up if
you double press the play/pause or menu buttons

Commit:
Update README.mdown

Jason-Morcos committed:
Added History support

Jason-Morcos committed:
Fixes several bugs

Jason-Morcos committed:
Fixed error handling. Modified url management

Jason-Morcos committed:
Updated .gitignore to ignore .xcuserstate files

Commit:
Update README.mdown

Commit:
Update README.mdown

Commit:
Update README.mdown

Commit:
Update README.mdown

Commit:
Update README.mdown

Jason-Morcos committed:
Added Favorites

Jason-Morcos committed:
Merge remote-tracking branch 'origin/master'

Conflicts:
	README.mdown

Jason-Morcos committed:
Improved textfield handling

Jason-Morcos committed:
Improved textfield handling. Added 100 item cap on history

jvanakker committed:
Merge pull request #1 from m-sch/master

added a script file for automatically changing the Availability.h

jvanakker committed:
Revert "added a script file for automatically changing the Availability.h"

jvanakker committed:
Merge pull request #4 from jvanakker/revert-1-master

Revert "added a script file for automatically changing the Availability.h"

jvanakker committed:
Merge pull request #3 from Moballo/master

Many, many features

Jason-Morcos committed:
Disable scroll view bounces

adaniello commented:
Great contribution!
:+1:

steventroughtonsmith commented:
Looks great!, but I won't be accepting pull requests in this repo. As per the readme: "This is about as complete as this project is ever going to get, so just treat it as sample code."

Jason-Morcos committed:
Added Usage Guide and improved Alert View management

Jason-Morcos committed:
Added missing period to Usage Guide

jvanakker committed:
Merge pull request #5 from Moballo/master

Master

patito committed:
Update README.mdown

jvanakker committed:
Merge pull request #6 from patito/patch-1

Update README.mdown

Jason-Morcos committed:
Merge remote-tracking branch 'jvanakker/master'

Jason-Morcos committed:
Export Control

Jason-Morcos committed:
Update Gitignore

mbonness committed:
Added a few menu options

Added menu options to increase/decrease font size, scale page to fit

Jason-Morcos committed:
Update local folder hierarchy

Jason-Morcos committed:
Merge branch 'dev'

Jason-Morcos committed:
Merge branch 'master' into stage

Jason-Morcos committed:
Merge branch 'master' into dev

jvanakker committed:
Merge pull request #8 from mbonness/patch-1

Added a few menu options

Jason-Morcos committed:
New project icon

Jason-Morcos committed:
Merge remote-tracking branch 'jvanakker/master' into dev

Jason-Morcos committed:
Merge branch 'dev'

Jason-Morcos committed:
Merge branch 'master' into stage

Jason-Morcos committed:
Merge branch 'stage'

Jason-Morcos committed:
Version 1.1

-Fixed scale pages to fit (clicking now works and setting is saved)
-Improved mobile/desktop mode (app no longer needs to restart)
-Updated user agent

Commit:
Merge pull request #1 from Moballo/dev

Version 1.1

Jason-Morcos committed:
Fixed readme

Jason-Morcos committed:
Merge branch 'dev'

Jason-Morcos committed:
Merge branch 'master' into dev

Jason-Morcos committed:
Merge branch 'dev'

Commit:
Update README.mdown

Jason-Morcos committed:
Merge branch 'master' into dev

Commit:
Update README.mdown

Commit:
Update README.mdown

Jason-Morcos committed:
Merge branch 'master' into dev

Jason-Morcos committed:
Remove requirement for Xcode file modification

Commit:
Merge pull request #2 from Moballo/dev

Release 2.0

Jason-Morcos committed:
Merge branch 'master' into dev

Jason-Morcos committed:
Merge branch 'master' into stage

Jason-Morcos committed:
Updated .gitignore

Jason-Morcos committed:
Merge branch 'stage' into dev

Jason-Morcos committed:
Updated .gitignore

Jason-Morcos committed:
Added Autoscroll to remove gray border

Jason-Morcos committed:
Merge pull request #4 from Moballo-LLC/dev

Release 2.1

Jason-Morcos committed:
Merge branch 'master' into dev

Jason-Morcos committed:
Patch Version for new Xcode

Jason-Morcos committed:
Merge pull request #6 from Moballo-LLC/dev

Release 3.0

Jason-Morcos committed:
Merge branch 'master' into dev

Jason-Morcos committed:
Update README.mdown

jvanakker committed:
Merge pull request #9 from Moballo-LLC/master

Version 1.1

Jason-Morcos committed:
Separate Offset Correction and Page Scaling

Jason-Morcos committed:
Merge pull request #7 from Moballo-LLC/dev

Release 3.0.1

Jason-Morcos committed:
4k Support

Jason-Morcos committed:
Merge branch 'master' into dev

* master:
  Update README.mdown

Jason-Morcos committed:
Merge pull request #8 from Moballo-LLC/dev

Release 4.0

Jason-Morcos committed:
Merge branch 'master' into dev

* master:

Commit:
Fix offset correction on launch

jvanakker committed:
Merge pull request #11 from thomasnordquist/fix-offset-correction-on-launch

Fix offset correction on launch

Commit:
Multiple improvements

Improved Apple HIG complyance, improved menu user experience, updated README as change to Availability.h is no longer required

Commit:
Updated Readme files after improvements

Commit:
New top Navigation bar added

jvanakker committed:
Update README.mdown

jvanakker committed:
Update README.mdown

jvanakker committed:
Update README.mdown

Jason-Morcos committed:
Merge pull request #10 from jvanakker/master

Bring over the great changes made on jvanakker's branch

Jason-Morcos committed:
Google should be https

Jason-Morcos committed:
Merge branch 'dev' of https://github.com/Moballo-LLC/tvOS-Browser into dev

* 'dev' of https://github.com/Moballo-LLC/tvOS-Browser:
  Update README.mdown
  Update README.mdown
  Update README.mdown
  New top Navigation bar added
  Updated Readme files after improvements
  Multiple improvements
  Fix offset correction on launch

Jason-Morcos committed:
Lots of bug fixes, some re-enabling of the menu button, and fixed some errors with text fields becoming uneditable

Jason-Morcos committed:
Added info to readme, renamed to Browser on home screen

Jason-Morcos committed:
Merge pull request #11 from Moballo-LLC/dev

Release 4.1

Jason-Morcos committed:
Add schema

Jason-Morcos committed:
Update repo readme

Jason-Morcos committed:
Fix using the full screen button not making top bar stay hidden

Jason-Morcos committed:
Merge pull request #12 from Moballo-LLC/dev

Release 4.2

Jason-Morcos committed:
Removing Compiled CocoaPods from git history
